draft-irtf-cfrg-vdaf.md

```diff
@@ -1964,9 +1968,14 @@ methods refer to constants enumerated in {{prio3-const}}.
 | `SHARES`          | in `[2, 256)`                                   |
 | `Measurement`     | `Flp.Measurement`                               |
 | `AggParam`        | `None`                                          |
-| `PrepState`       | `tuple[Vec[Flp.Field], Optional[Bytes]]`        |
-| `OutShare`        | `Vec[Flp.Field]`                                |
+| `PublicShare`     | `Optional[list[bytes]]`                         |
+| `InputShare`      | `tuple[Union[tuple[list[Flp.Field],list[Flp.Field]],tuple[bytes, bytes]],Optional[bytes]]` |
```

Comment:
For readability, but I'm not sure if Markdown or the RFC renderer will like it:
```suggestion
| `InputShare`      | `tuple[`
|                   | `   Union[`
|                   | `       tuple[list[Flp.Field],list[Flp.Field]],`
|                   | `       tuple[bytes, bytes],`
|                   | `   ],`
|                   | `   Optional[bytes],`
|                   | `]`
```

Reply:
RFC tools seem to hate this unforutnately.